cumulus_library/study_parser.py

```diff
@@ -86,13 +86,25 @@ def get_study_prefix(self) -> Optional[str]:
         """
         return self._study_config.get("study_prefix")
 
-    def get_sql_file_list(self) -> Optional[StrList]:
+    def get_sql_file_list(self, continue_from: str = None) -> Optional[StrList]:
         """Reads the contents of the sql_config array from the manifest
 
         :returns: An array of sql files from the manifest, or None if not found.
         """
         sql_config = self._study_config.get("sql_config", {})
-        return sql_config.get("file_names", [])
+        sql_files = sql_config.get("file_names", [])
+        if continue_from:
+            match_found = False
+            for file in sql_files:
+                print(continue_from.replace(".sql", ""))
+                print(file.replace(".sql", ""))
+                if continue_from.replace(".sql", "") == file.replace(".sql", ""):
+                    sql_files = sql_files[sql_files.index(file) :]
+                    match_found = True
+                    break
+            if not match_found:
+                sys.exit(f"No tables matching '{continue_from}' found")
```

Comment:
You ready for this Python nonsense? Just do `else:` and skip `match_found`. [For/else](https://docs.python.org/3/tutorial/controlflow.html#break-and-continue-statements-and-else-clauses-on-loops) lets you run code if the for loop never broke.

i.e.:
```
for file in sql_files:
    if continue_from.replace(".sql", "") == file.replace(".sql", ""):
        sql_files = sql_files[sql_files.index(file) :]
        break
else:
    sys.exit(f"No tables matching '{continue_from}' found")
```

Alternatively, you could also do `return sql_files[sql_files.index(file) :]` and unconditionally exit after the loop.

Or keep it! Whatever works for your sense of code cleanliness.

Reply:
taking the `for\else` approach, just in case there's other post loop stuff that might end up here in the future.